staging/src/k8s.io/apimachinery/pkg/api/meta/restmapper.go

```diff
@@ -24,6 +24,8 @@ import (
 
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/gengo/namer"
```

Comment:
gengo is for code-generation prior to build, not something we want to rely on at runtime

Reply:
I see. What suggestion do you have on this one? `namer` ability to provide plural names was suggested in https://github.com/kubernetes/kubernetes/pull/110053#issuecomment-1585644795 as a replacement for `UnsafeGuessKindToResource`. What other options do we have?